frontend/src/pages/DetailedCourse/styled.ts

```diff
@@ -1,4 +1,4 @@
-import styled from 'styled-components';
+import { styled } from '@mui/material';
 import { Box } from '@mui/system';
```

Comment:
in some files, Box is imported from mui/system, and in others, mui/material. can you fix it too please?

Reply:
done